chromium_src/services/network/restricted_cookie_manager.h

```diff
@@ -14,6 +14,8 @@
 
 #define RemoveChangeListener                                             \
   NotUsed() const {}                                                     \
+  base::Time ModifyExpiration(const base::Time& expiry_date,             \
```

Comment:
there's no need to make this a method on the class

Reply:
No strong feelings on my end, just that it seemed cleanest to keep this method together with the existing ones?